deployments/liqo/values.yaml

```diff
@@ -137,7 +137,7 @@ auth:
     port: ""
     class: ""
   config:
-    allowEmptyToken: true
+    allowEmptyToken: false
```

Comment:
Could you please add a brief description to be rendered in the helm documentation about what concerning the empty token?